.github/workflows/code-linting.yml

```diff
@@ -1,4 +1,5 @@
-name: Code Linting
+name: pre-commit
+
```

Comment:
Should changes in this file be in this PR?

Reply:
yeah